docs/content/AvatarStack.mdx

```diff
@@ -26,18 +26,10 @@ AvatarStack is used to display more than one Avatar in an inline stack.
 </AvatarStack>
 ```
 
-## System props
-
-<Note variant="warning">
-
-System props are deprecated in all components except [Box](/Box). Please use the [`sx` prop](/overriding-styles) instead.
-
-</Note>
-
-AvatarStack components get `COMMON` system props, AvatarStack.Item components do not get any system props. Read our [System Props](/system-props) doc page for a full list of available props.
-
 ## AvatarStack Component props
 
-| Name       | Type    | Default | Description                       |
-| :--------- | :------ | :-----: | :-------------------------------- |
-| alignRight | Boolean |         | Creates right aligned AvatarStack |
+| Name       | Type              | Default | Description                          |
+| :--------- | :---------------- | :-----: | :----------------------------------- |
+| alignRight | Boolean           |         | Creates right aligned AvatarStack    |
+| children   | React.Node        |         | The avatars                          |
```

Comment:
Non-blocking: Do we typically include `children` (since every? React component supports it)?

Reply:
Doesn't seem like we generally do. Would it be useful to include it in places where we have an expectation of a certain kind of component? e.g.,:

```
| children   | React.Node        |         | Generally an `Avatar`     |
```

Reply:
Does the `<Props>` component work for this component?

Reply:
Good question — I'll check.

Reply:
Yep, will switch over to that.